requirements.txt

```diff
@@ -8,5 +8,10 @@ lxml==4.6.2
 pygments==2.8.0
 python-dateutil==2.8.1
 pyyaml==5.4.1
-requests[socks]==2.25.1
+httpx[http2]==0.17.1
+brotlipy==0.7.0
+uvloop==0.15.2; python_version >= '3.7'
+uvloop==0.14.0; python_version < '3.7'
```

Comment:
I'm not sure if this version should be included.
There are multiple bug fixes in the releases 0.15.* : 
* https://github.com/MagicStack/uvloop/releases